frontend/src/features/properties/components/forms/subforms/columns.tsx

```diff
@@ -100,9 +118,15 @@ export const getAssessedCols = (
     },
   ];
   if (includeImprovements) {
-    return [{ Header: 'Assessed Value', columns: [...basicAssessed, ...improvements] }];
+    return [
+      {
+        Header: generateAssessedHeader(),
+        id: 'assessed',
```

Comment:
Soon as I changed the header to something other than strictly text it would complain there was no `id` so I just added in an id for both areas and everything now behaves as expected..

Reply:
Interesting

Reply:
it may use the text under the hood to generate an id.

Reply:
Ahhh good point @devinleighsmith